lib/functionTemplates/template.js

```diff
@@ -0,0 +1,28 @@
+const format = require('string-format');
+
+const node8Template =
+`'use strict'
+
+/**
+ * @type {{ import("./binaris").Handler }}
+ */
+exports.{entrypoint} = async (body, context) => {
+  const name = context.request.query.name || body.name || 'World';
+  return \`Hello \${{name}}!\`;
+};`;
+
+const pythonTemplate =
+`def {entrypoint}(body, req):
+    name = req.query.get('name') or body.get('name') or 'World'
```

Comment:
This template is wrong, the second command is the context, accessing `context.query` is deprecated.

Reply:
> This template is wrong, the second command is the context, accessing `context.query` is deprecated.

If you don't solve here, please open an issue.